MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -6242,6 +6242,12 @@ public string GetProgressiveAudioFullCommandLine(EncodingJobInfo state, Encoding
                 audioTranscodeParams.Add("-acodec " + GetAudioEncoder(state));
             }
 
+            if (GetAudioEncoder(state).StartsWith("pcm_", StringComparison.Ordinal))
+            {
+                audioTranscodeParams.Add("-f " + GetAudioEncoder(state).Substring(4));
```

Comment:
```suggestion
                audioTranscodeParams.Add(string.Concat("-f ", GetAudioEncoder(state).AsSpan(4)));
```
Sorry the compiler seems to insist on this

Reply:
I just performed a force push to remove the faulty build from the commit history after discovering a CI failure. However, this patch is based on the outdated version.

Would it be possible for you to review it again? Alternatively, I can create a new commit to implement these changes, but it will result in a change of the authorship.